07.bowling_object/lib/frame.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class Frame
+  def initialize(first, second, third)
+    @first  = first
+    @second = second
+    @third  = third
```

Comment:
```suggestion
  def initialize(shots)
    @shots = shots
```

のように配列で保持するようにすれば、

```ruby
@shots.sum(&:point)
```

のようにして合計値を求めたりすることができそうです。